pgo/cmd/auth.go

```diff
@@ -55,7 +59,7 @@ func StatusCheck(resp *http.Response) {
 	}
 }
 
-func UserHomeDir() string {
+func userHomeDir() string {
```

Comment:
Great change! Please document this function if we're going to be renaming it ;-)

Reply:
Function is now documented.